metagenomics/diamond_dbs/README.md

```diff
@@ -0,0 +1,3 @@
+# Test Diamond Databases
+
+The file `test-cheese-proteins-db.dmnd` is a small DIAMOND database made from proteins from the subsetted cheese assemblies that are used as test inputs to the `Arcadia-Science/metagenomics` workflow, from both the Illumina and Nanopore samples. The database was created by predicting proteins from these subsetted sample files using `prodigal` from the workflow, concatenating them together, and creating a DIAMOND database with `diamond makedb --in concatenated_cheese_proteins.fasta --db test-cheese-proteins-db.dmnd`.
```

Comment:
Clever! did you add taxonomy as well? do you need it for this?

Reply:
for this test conf you won't need taxonomy in it